src/infi/clickhouse_orm/database.py

```diff
@@ -79,6 +81,56 @@ def gen():
                 yield ('\n'.join(batch) + '\n').encode('utf-8')
         self._send(gen())
 
+    def insert_try_best(self, model_instances, batch_size = 1000, is_create = False, insert_count = 0, first_recu = True):
```

Comment:
What is this method, why is it needed, what kind of problems does it try to solve?